flask/config.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, name, get_converter=None):
         self.__name__ = name
         self.get_converter = get_converter
 
-    def __get__(self, obj, type=None):
+    def __get__(self, obj):
```

Comment:
why that change? i roughly recall the second parameter is part of the descriptor protocol